### ISSUE TITLE
.get returns null

### ISSUE THREAD

**Issue by silentlight** [revolver-app/vuex-orm#21]
Hello again @kiaking 

I have a question regarding `.get()` method. In an example like below:
```
records() {
  return this.$store.getters['entities/timesheets/query']()
          .where(record => record.status === 'submitted' || record.status === 'resubmitted')
          .get();
}
```
If no records are found we receive `null`. So to pass it to `v-for` or pass it to component as a required prop I have to first check if it is not null. Wouldn't it be more intuitive if `.get()` returned empty array `[]` if no records were found? I am coming from Rails world and that's what I am used to ;) Let me know what you think?

**Comment by kiaking** [comment]
Hi @silentlight 

Thank you for your advice. I've checked Laravel as well, and it does return `[]` (empty array). So yeah I think you got the point.

I'll make this change!

**Comment by silentlight** [comment]
Sounds good @kiaking :)

**Comment by timoschwarzer** [comment]
A workaround is to append `|| []` to your expression like this:
`const users = getters['entities/users/all']() || []`
Then `users` will be an empty array when there are no users.

**Comment by silentlight** [comment]
Good point @timoschwarzer. That's exactly what I am doing right now, but as you said it would be nice if that was a default option :)

**Comment by kiaking** [comment]
Hi forks!

This issue is now fixed with Vuex ORM 0.11.5. It will now return empty array instead of null.

If you were doing

```js
const users = getters['entities/users/all']() || []
```

as a workaround, this update will not break them, but you are now free to remove `|| []` part from your code 👍 

Thank you all for your support!

**Comment by timoschwarzer** [comment]
That's great! Thanks! :smile:
